packages/wasm/src/build.ts

```diff
@@ -59,6 +59,6 @@ export const buildActionParallel = async (
 const loadProvingKey = async (actionType: ActionType) => {
   const keyType = provingKeys[actionType];
   if (!keyType) return;
-  const keyBin = (await fetch(`bin/${actionType}_pk.bin`)).arrayBuffer();
+  const keyBin = (await fetch(`bin/${keyType}_pk.bin`)).arrayBuffer();
```

Comment:
Fixes a bug where the proving key wasn't being fetched correctly. We didn't notice this earlier because `actionType` and `keyType` are the same for the more common actions, but I noticed it while working on undelegation claims because the `undelegateClaim` action uses a `convert` key.

Reply:
oh! Good catch.